src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -32,37 +58,86 @@
     (range (rand-int 50))))
 
 
+(defn render-import-modal [state props refs]
+  [comps/ModalDialog
+   {:width 600
+    :content (react/create-element
+               [:div {}
+                [:div {:style {:backgroundColor "#fff"
+                               :borderBottom (str "1px solid " (:line-gray style/colors))
+                               :padding "20px 48px 18px"
+                               :fontSize "137%" :fontWeight 400 :lineHeight 1}}
+                 "Import a Method Configuration"]
+                [:div {:style {:padding "22px 48px 40px"
+                               :backgroundColor (:background-gray style/colors)}}
+                 "Destination Name : "
+                 (style/create-text-field
+                   {:defaultValue (get-in @state [:selected-conf "name"])
+                    :ref "destinationName"})
+                 [:br]
+                 "Destination Namespace : "
+                 (style/create-text-field
+                   {:defaultValue (get-in @state [:selected-conf "namespace"])
+                    :ref "destinationNamespace"})
+                 [:br]
+                 [comps/Button
+                  {:title-text "Import configuration"
+                   :icon :plus
+                   :onClick
+                   (fn []
+                     (let [workspace (:workspace props)
+                           dest-name-value (.-value
+                                             (.getDOMNode (@refs "destinationName")))
+                           dest-namespace-value (.-value (.getDOMNode
+                                                           (@refs "destinationNamespace")))
+                           dest-workspace-name (get workspace "name")
+                           dest-workspace-namespace (get workspace "namespace")
+                           methodRepoQuery (create-post-data
+                                             (:selected-conf @state)
+                                             dest-name-value
+                                             dest-namespace-value
+                                             dest-workspace-name
+                                             dest-workspace-namespace)
+                           url-method :post
```

Comment:
Same as my previous comment.  Same also for path and data below at least, if not all of the ajax parameters.